packages/msw-addon/src/index.new.ts

```diff
@@ -0,0 +1,3 @@
+export { initialize, InitializeOptions } from '@build-time/initialize'
```

Comment:
🤯

Reply:
Note: it's just a fancy name to make it clear where this import will be resolved to. This import identifier is an alias in `tsup.confug.ts` that resolves it to different modules (`initialize.browser.ts`, `initialize.node.ts`, etc) on build time.